### ISSUE TITLE
Grabbed objects with physics stutter badly in the demo

### ISSUE THREAD

**Issue by Klendat** [provencher/MRTK-Quest#96]
Objects that have rigidbody physics (the coffee cup, the cubes etc.) stutter badly when they're grabbed and moved. The stuttering disappears when I let go of the object and it just falls to the floor smoothly.

The objects that don't have any physics are fine when grabbed. The floaty coffee cup that doesn't have gravity moves smoothly when manipulated.

Interestingly enough it isn't very noticeable in recordings which makes me think this is some sort of framerate issue

I just downloaded MRTK-Quest_v120.apk from the releases and ran it on my quest 2 (v23) so nothing was changed on my end.

**Comment by provencher** [comment]
Hmm I'll report this to the MRTK folks. I believe it's a problem with the Object Manipulator's logic around physics, along with the Oculus V23 activating 90 Hz across the board.

Could you try changing the fixed timestep time in your Unity project settings to 1/90, to see if the issue persists? 

Cheers!

**Comment by Klendat** [comment]
> Could you try changing the fixed timestep time in your Unity project settings to 1/90, to see if the issue persists?
> 

Yeah that seems to be the issue. I changed the fixed timestep to 0.011 and now the grabbed object motion is smooth.
